types/got/index.d.ts

```diff
@@ -98,18 +104,23 @@ declare namespace got {
         json?: boolean;
     }
 
-    interface TimoutRequestOptions extends http.RequestOptions {
+    interface RequestOptions extends http.RequestOptions {
```

Comment:
This is potentially breaking change, but I think it's not a problem for major release of definition.